diff -up libtool-2.2.6/configure.echo libtool-2.2.6/configure
--- libtool-2.2.6/configure.echo	2008-12-03 09:26:03.000000000 -0500
+++ libtool-2.2.6/configure	2008-12-03 09:26:14.000000000 -0500
@@ -27606,7 +27606,7 @@ objdir=$objdir
 SHELL=$lt_SHELL
 
 # An echo program that does not interpret backslashes.
-ECHO=$lt_ECHO
+echo=$lt_ECHO
 
 # Used to examine libraries when file_magic_cmd begins with "file".
 MAGIC_CMD=$MAGIC_CMD
