--- libtool-1.5.22/libltdl/ltdl.c.relativepath	2007-02-08 09:20:54.000000000 -0500
+++ libtool-1.5.22/libltdl/ltdl.c	2007-02-08 09:46:41.000000000 -0500
@@ -3203,9 +3203,10 @@
 	    }
 #endif
 	}
-      if (!file)
+	if (!file)
 	{
-	  file = fopen (filename, LT_READTEXT_MODE);
+	  if(strstr(filename,"/") || strcmp((filename + strlen(filename) - 3), ".la") != 0)
+	    file = fopen (filename, LT_READTEXT_MODE);
 	}
 
       /* If we didn't find the file by now, it really isn't there.  Set
